reversal algorithm

Sapna2001 opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
Yes

### Description

(Write your answer here.)
Reversal algorithm to rotate an array in java under the algorithms section.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
#3640

Sapna2001 committed:
reversal algorithm

Sapna2001 commented:
Replit link : https://replit.com/@Sapna2001/reversal

HarshCasper commented on Java/Algorithms/reversalAlgorithm.java:
```diff
@@ -0,0 +1,60 @@
+public class ReversalAlgorithm {
```
Add a beginner friendly description

HarshCasper commented on Java/Algorithms/reversalAlgorithm.java:
```diff
@@ -0,0 +1,60 @@
+	}
+
+	public static void main(String[] args) {
+		int[] array = {1,2,3,4,5,6,7};
```
Use dynamic input

HarshCasper commented on Java/Algorithms/reversalAlgorithm.java:
```diff
@@ -0,0 +1,60 @@
+
+// Output:
+// 3 4 5 6 7 1 2 -> after left rotation of original array
+// 1 2 3 4 5 6 7 -> after right rotation of left rotated array
```
Put in multi-line comments

HarshCasper replied:
Add a new line at the EOF

Sapna2001 committed:
merge master
Merge branch 'master' of https://github.com/TesseractCoding/NeoAlgo into reversalAlgorithm

Sapna2001 committed:
reversal algorithm modified

Sapna2001 committed:
reversal algorithm modified

Sapna2001 commented:
@HarshCasper I have made the changes

Sapna2001 committed:
Merge branch 'master' of https://github.com/TesseractCoding/NeoAlgo into reversalAlgorithm

ankitaggarwal23 reviewed (commented):
Code plagiarized from [here](https://www.geeksforgeeks.org/program-for-array-rotation-continued-reversal-algorithm/).
